Automattic/wp-calypso
# WooCommerce: Add Dimensions Input Component

This adds a general reusable `FormDimensionsInput` component. It provides a wrapper for width, length, and height inputs.

I am putting this component under the`woocommerce` directory for now. We can always split it out into Calypso's form components if we feel it is useful outside of WC, and if we feel like we don't need to make additional changes to it.

This PR also adds the start of the delivery details card, to provide a functioning example of the dimensions input. This card is to configure the dimensions of a non-variable product. 

#13698 will use this for variation data.
See #13665.

To test:
* Go to `http://calypso.localhost:3000/store/products/:site/add`.
* Type in each of the dimension inputs and make sure you can update their values.

Delivery Details Preview (Other fields to come later):

<img width="759" alt="screen shot 2017-05-09 at 11 28 24 am" src="https://cloud.githubusercontent.com/assets/689165/25866250/a86fef62-34aa-11e7-9d6c-8a317f615c70.png">
